src/main/java/com/google/devtools/build/lib/hash/Blake3JNI.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.devtools.build.lib.hash;
+
+import com.google.devtools.build.lib.jni.JniLoader;
+import java.nio.ByteBuffer;
+
+public class Blake3JNI {
```

Comment:
Not sure if there are some JNI quirks with reflection and/or visibility, but perhaps this should be `final class` instead of `public class`

Reply:
Done.